src/RestControllers/AppointmentRestController.php

```diff
@@ -65,20 +122,39 @@ public function getAllForPatient($pid)
         return RestControllerHelper::responseHandler($serviceResult, null, 200);
     }
 
-    public function post($pid, $data)
+    /**
+     * Creates a new appointment for a patient identified by UUID.
+     *
+     * @param string $puuid The UUID of the patient for whom the appointment is being created.
+     * @param array $data An array containing the appointment data.
+     * @return mixed Returns the result of the appointment creation operation.
+     */
+    public function post(string $puuid, array $data)
     {
-        $data['pid'] = $pid;
+        $data['puuid'] = $puuid;
+        $data['pc_time'] = date("Y-m-d H:i:s");
         $validationResult = $this->appointmentService->validate($data);
-
-        $validationHandlerResult = RestControllerHelper::validationHandler($validationResult);
+        $validationHandlerResult = RestControllerHelper::validationHandler($validationResult);        
         if (is_array($validationHandlerResult)) {
             return $validationHandlerResult;
         }
 
-        $serviceResult = $this->appointmentService->insert($pid, $data);
+        $serviceResult = $this->appointmentService->insert($puuid, $data);
         return RestControllerHelper::responseHandler(array("id" => $serviceResult), null, 200);
     }
 
+    /**
+     * Deletes an appointment by its UUID.
+     *
+     * @param string $euuid The UUID of the appointment to delete.
+     * @return mixed Returns the result of the appointment deletion operation.
+     */
+    public function deleteByUuid(string $euuid)
```

Comment:
- if is possible add the return type:
public function deleteByUuid(string $euuid): TypeToReturn

- What thats menas $euuid? If is possible create a more descriptive name [Ex: $element_uuid]